s3/sc-s3-synapse-ra.yaml

```diff
@@ -70,6 +70,14 @@ Resources:
       OwnershipControls:
         Rules:
           - ObjectOwnership: BucketOwnerPreferred
+      LifecycleConfiguration:
+        Rules:
+          - Id: IntelligentTieringRule
```

Comment:
I would call this IntelligentTieringClassTransitionRule

Reply:
done